content/workshops/How to be a great Mentor/_index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
There cannot be spaces or capitalisation in the link, it will break things:
content/workshops/How to be a great Mentor/_index.md